cmd/root.go

```
@@ -79,14 +85,17 @@ func (a *MethodAws) InitRootCommand() {
 				outputFilePointer = nil
 			}
 			a.OutputConfig = writer.NewOutputConfig(outputFilePointer, format)
-			a.RootFlags.Regions = common.GetAWSRegions(a.RootFlags.Regions)
-			cmd.SetContext(svc1log.WithLogger(cmd.Context(), config.InitializeLogging(cmd, &a.RootFlags)))
-			awsconfig, err := awsconfig.LoadDefaultConfig(cmd.Context())
+
+			a.RootFlags.Regions, err = common.GetAWSRegions(cmd.Context(), *a.AwsConfig, a.RootFlags.Regions)
 			if err != nil {
-				return err
+				a.OutputSignal.ErrorMessage = aws.String(err.Error())
+			}
+
+			cmd.SetContext(svc1log.WithLogger(cmd.Context(), config.InitializeLogging(cmd, &a.RootFlags)))
+
+			if len(a.RootFlags.Regions) > 0 {
```

Do we want to throw an error if Regions == 0?